trendmicro_apex/help.md

```diff
@@ -0,0 +1,134 @@
+# Description
+
+Trend Micro Apex offers modern advanced automated threat detection and response.  Apex agents have more than antivirus
+capabilities, they are an extension of the Apex threat management system.  
+
+# Key Features
+
+* Reporting suspicious ips, urls and other similar content
+* Reporting suspicious files and their contents
+
+# Requirements
+
+* A host running Apex Security Manager
```

Comment:
Cloud API URL?

Reply:
The API's are the same for both the on-premise and Apex One as a Service offering so we can call both of those out so users know.

Reply:
I made it say API cloud url in the requirements, and mentioned SaaS/onpremise in the description of the plugin.